.github/workflows/build-test.yaml

```diff
@@ -206,36 +206,91 @@ jobs:
       - name: Run K3s and master in it
         run: tests/run-master-in-k3s.sh local/freeipa-server:${{ matrix.os }} freeipa-server-${{ matrix.os }}.tar
 
+  push-after-success-precondition:
+    name: Check pipeline settings
+    needs: [ test-docker, test-podman, test-rootless-podman, test-upgrade, test-k3s ]
+    runs-on: ubuntu-20.04
+    env:
+      DOCKER_TARGET_LIST: "${{ secrets.DOCKER_TARGET_LIST }}"
+      DOCKER_CREDENTIALS_FILE: "${{ secrets.DOCKER_CREDENTIALS_FILE }}"
```

Comment:
We likely want to avoid using the `DOCKER_`, especially now that the code is not specifically about pushing to Docker Hub.

Reply:
Thanks! Changed.